config/Mekanism/common.toml

```diff
@@ -3,10 +3,10 @@
 [common]
 	#Displayed energy type in Mekanism GUIs and network reader readings.
 	#Allowed Values: JOULES, FORGE_ENERGY
-	energyType = "FORGE_ENERGY"
+	energyType = "JOULES"
```

Comment:
I am fine with this, but the reason we initially changed it is that it just caused confusion for players with it having a different name